packages/tools/examples/segmentBidirectionalTool/index.ts

```diff
@@ -0,0 +1,465 @@
+import {
+  RenderingEngine,
+  Types,
+  Enums,
+  setVolumesForViewports,
+  volumeLoader,
+  getEnabledElement,
+} from '@cornerstonejs/core';
+import * as cornerstoneTools from '@cornerstonejs/tools';
+
+import {
+  initDemo,
+  createImageIdsAndCacheMetaData,
+  setTitleAndDescription,
+  addDropdownToToolbar,
+  addSliderToToolbar,
+  setCtTransferFunctionForVolumeActor,
+  addButtonToToolbar,
+} from '../../../../utils/demo/helpers';
+
+// This is for debugging purposes
+console.warn(
+  'Click on index.ts to open source code for this example --------->'
+);
+
+const {
+  SegmentationDisplayTool,
+  ToolGroupManager,
+  Enums: csToolsEnums,
+  segmentation,
+  RectangleScissorsTool,
+  SphereScissorsTool,
+  CircleScissorsTool,
+  BrushTool,
+  PaintFillTool,
+  PanTool,
+  ZoomTool,
+  StackScrollTool,
+  StackScrollMouseWheelTool,
+  BidirectionalTool,
+  utilities: cstUtils,
+} = cornerstoneTools;
+
+const { roundNumber } = cstUtils;
+
+const { MouseBindings, KeyboardBindings } = csToolsEnums;
+const { ViewportType } = Enums;
+const { segmentation: segmentationUtils } = cstUtils;
+let renderingEngine;
+const viewportId = 'CT_AXIAL';
```

Comment:
Could you enhance this example by adding two side-by-side viewports? On the left, we'll have the stack viewport CT, and on the right, the volume (acquisition) PT or something similar. If we also have support for reconstructions, perhaps we can make the right one SAG PT; otherwise, it's alright to keep it as just the acquisition volume. The purpose behind this is to ensure that we test on stack seg from now on for each scenario.